ndspy/bmg.py

```diff
@@ -366,15 +373,15 @@ class Escape:
         type and optional parameter data.
         """
 
-        def __init__(self, type=0, data=b''):
+        def __init__(self, type: int = 0, data: bytes = b'') -> None:
             self.type = type
             self.data = data
```

Comment:
Can you please add declarations for these (with type annotations) to the class body?